histogram.js

```diff
@@ -118,14 +120,30 @@ class HistogramElement extends HTMLElement {
         this._histo.update(d);
       });
     }
-    else {
-      (async () => {
-        const data = await getDataFromAttr(this);
-        if (data) {
-          this._histo.update(data);
+
+    const renderHistogramWhenVisible = () => {
+        if (!useBroker) {
+          (async () => {
```

Comment:
You can simplify this a bit by making `renderHistogramWhenVisible` async and removed the inline function